data/json/items/comestibles/bread.json

```
@@ -289,23 +297,29 @@
     "id": "toad_in_a_hole",
     "name": { "str_sp": "toad in a hole" },
     "copy-from": "toast",
-    "description": "A cooked egg served inside a piece of toast.",
+    "description": "A cooked egg served inside a piece of toast, not to be confused with the english recipe of the same name.",
+    "extend": { "flags": [ "EATEN_HOT", "FREEZERBURN" ] },
+    "material": [ "wheat", "egg" ],
     "relative": { "calories": 80, "fun": 4, "flags": [ "FREEZERBURN" ], "vitamins": [ [ "iron", 4 ], [ "calcium", 3 ] ] }
   },
   {
     "type": "COMESTIBLE",
     "id": "toad_in_a_hole_wheat_free",
     "name": { "str_sp": "wheat-free toad in a hole" },
     "copy-from": "toast_wheat_free",
-    "description": "A cooked egg served inside a piece of wheat-free toast.",
-    "relative": { "calories": 80, "fun": 4, "flags": [ "FREEZERBURN" ], "vitamins": [ [ "iron", 4 ], [ "calcium", 3 ] ] }
+    "description": "A cooked egg served inside a piece of wheat-free toast, , not to be confused with the english recipe of the same name.",
```

There's an extra comma there, and English should be capitalized.
```suggestion
    "description": "A cooked egg served inside a piece of wheat-free toast, not to be confused with the English recipe of the same name.",
```